docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -329,7 +333,7 @@ public class OidcTokenRequestCustomizer implements OidcRequestFilter {
 <1> Get `OidcConfigurationMetadata` which contains all supported OIDC endpoint addresses.
 <2> Use `OidcConfigurationMetadata` to filter requests to the OIDC token endpoint only.
```

Comment:
```suggestion
<2> Use `OidcConfigurationMetadata` to only filter requests to the OIDC token endpoint.
```

Reply:
@MichalMaler, I think your proposed update changes the intended meaning here, that is, the original suggests that requests can be filtered to the "OIDC token endpoint" only (and not to any other type of endpoint), whereas, your changes state that OidcConfigurationMetadata can only "requests" (and not any other type of data) to the OIDC token endpoint.
@sberyozkin, which is the intended meaning here?

Reply:
Thanks @sberyozkin. Shall keep it as it is then. Thank you.